### PR TITLE
Feature module1 reflow

### PR DESCRIPTION
**Pull request by diaza**
Module1 reflow, and makes proto_nd_flow scripts work on data.

### CODE REVIEW

File: src/proto_nd_flow/reco/charge/calib_prompt_hits.py [hunk 1]
```diff
@@ -152,14 +155,18 @@ def run(self, source_name, source_slice, cache):
...
             index_arr = packets_index.data[mask]
         else:
             n = 0
             index_arr = np.zeros((0,), dtype=packets_index.dtype)
 
-        has_mc_truth = packet_seg_bt is not None
+        if resources['RunData'].is_mc:
+            has_mc_truth = packet_seg_bt is not None
+        else:
+            has_mc_truth = False
```
Comment (krwood) [inline src/proto_nd_flow/reco/charge/calib_prompt_hits.py:169]: At this point `has_mc_truth` is redundant with `resources['RunData'].is_mc`

File: src/proto_nd_flow/reco/light/wvfm_align.py
Comment (krwood) [inline src/proto_nd_flow/reco/light/wvfm_align.py:1]: I would defer this to @liviocali

**Review by krwood — APPROVED** [review]

File: src/proto_nd_flow/reco/charge/calib_prompt_hits.py (hunk 1 above)
Comment (diaza) [reply src/proto_nd_flow/reco/charge/calib_prompt_hits.py:169]: Is there no weird situation where it's MC but without backtracking packets?

File: src/proto_nd_flow/reco/light/wvfm_align.py
Comment (liviocali) [reply src/proto_nd_flow/reco/light/wvfm_align.py:1]: We need to test this with 2x2 larndsim inputs.

Comment (diaza) [reply src/proto_nd_flow/reco/light/wvfm_align.py:1]: MC doesn't go through this part of the if-else statement. I changed this line to accommodate the change that had been made in the tai_ns shape, from (adc, sipm) to (adc)

File: data/module1_flow/module0.yaml
Comment (YifanC) [inline data/module1_flow/module0.yaml:1]: Shall we name it `module1.yaml`?

File: src/proto_nd_flow/reco/light/wvfm_align.py
Comment (YifanC) [inline src/proto_nd_flow/reco/light/wvfm_align.py:1]: dumb question, what's the reason for this change?

File: src/proto_nd_flow/reco/light/adc64_event_generator-single_module.py
Comment (YifanC) [inline src/proto_nd_flow/reco/light/adc64_event_generator-single_module.py:1]: Sorry that I didn't understand the details. Why it cannot be merged with [adc64_event_generator.py](https://github.com/DUNE/ndlar_flow/blob/322d6103f50b2faedfa8bc2d42681652a07f6396/src/proto_nd_flow/reco/light/adc64_event_generator.py)?

File: yamls/module1_flow/reco/charge/CalibHitBuilder.yaml
Comment (YifanC) [inline yamls/module1_flow/reco/charge/CalibHitBuilder.yaml:1]: I'm also curious why it is set to 10 while the simulation version set it to 200. I'd imagine this parameter should be set the same?

File: data/module1_flow/module0.yaml
Comment (diaza) [reply data/module1_flow/module0.yaml:1]: Sure

File: src/proto_nd_flow/reco/light/wvfm_align.py
Comment (diaza) [reply src/proto_nd_flow/reco/light/wvfm_align.py:1]: At some point the shape of tai_ns shaped from (n_adcs, n_channels) to (n_adcs), and this change accommodates for that. The change in dimensions had probably only been tested on mc, and this line of code is for data.

File: src/proto_nd_flow/reco/light/adc64_event_generator-single_module.py
Comment (diaza) [reply src/proto_nd_flow/reco/light/adc64_event_generator-single_module.py:1]: My understanding from @liviocali  is that the file format for the 2x2 is different than that used for the single modules

File: yamls/module1_flow/reco/charge/CalibHitBuilder.yaml
Comment (diaza) [reply yamls/module1_flow/reco/charge/CalibHitBuilder.yaml:1]: Do you mean [here](https://github.com/DUNE/ndlar_flow/blob/1ff5456cb04d3b2e4ced50dacc358310f9f31191/yamls/proto_nd_flow/reco/charge/CalibHitBuilder.yaml) for the simulation version?
